cms/templates/js/mock/mock-container-paged-xblock.underscore

```diff
@@ -13,39 +13,6 @@
     <div class="xblock" data-locator="locator-container" data-request-token="page-render-token"
          data-init="MockXBlock" data-runtime-class="StudioRuntime" data-runtime-version="1">
 
-         <script type="text/template" id="paging-header-tpl">
```

Comment:
These are the inlined templates that are now loaded via RequireJS text dependencies. I think it was a mistake that they were included in the mock files (it was probably all just copied in from a rendered page).